sql/sql_lex.cc

```diff
@@ -2115,6 +2115,7 @@ void st_select_lex::init_query()
   item_list.empty();
   join= 0;
   having= prep_having= where= prep_where= 0;
+  saved_conds= 0;
```

Comment:
In C++ NULL == 0 and some people prefer to use 0 for code conforming to standard older than C++03.